React/finger-exercises/ecmascript/01 - spread/utils.js

```diff
@@ -0,0 +1,9 @@
+// This have a linter problem, it says that we should prefer
+// export default
+// Would you like to fix this issue ?
+
+export function isArray(aPossibleArray) {
+  return aPossibleArray.constructor === Array;
+}
+
+// export default isArray;
```

Comment:
Did you checkout the code comment above? It gives you a hint on what to do here. 😉